pubtools/_quay/iib_operations.py

```diff
@@ -105,19 +105,19 @@ def task_iib_add_bundles(
     )
     # Index image used to fetch manifest list. This image will never be overwritten
     iib_namespace = build_details.index_image_resolved.split("/")[1]
-    image_digest = build_details.index_image_resolved.split("@")[1]
-    intermediate_index_image = image_schema_digest.format(
+    iib_repo = build_details.index_image_resolved.split("/")[2].split("@")[0]
+    permanent_index_image = image_schema_tag.format(
         host=target_settings.get("quay_host", "quay.io").rstrip("/"),
         namespace=iib_namespace,
-        repo="iib",
-        digest=image_digest,
+        repo=iib_repo,
```

Comment:
build_tags are added to intermediate index image right now. It seems an issue of IIB. I raised a question in https://code.engineering.redhat.com/gerrit/c/rcm-pub/+/289084.

Reply:
@emilyzheng Is this still an issue?

Reply:
Yeah, it's updated to get unique tag from "iib" in pub https://code.engineering.redhat.com/gerrit/c/rcm-pub/+/304487

Reply:
I'm not sure I understand. Is it still an issue in pubtools-quay workflow?

Reply:
It is. The unique tag is in repo "iib", but not in the repo parsed from index_image_resolved. So this line seems an unnecessary change.

Reply:
I changed the repo to "iib"